e2e/game.js

```diff
@@ -3,74 +3,34 @@ import { Selector, ClientFunction } from 'testcafe';
 import MainPage from './main_page.js';
 import GamePage from './game_page.js';
 
```

Comment:
This file is for tests, isn't it? Yet it is not in a test-named folder nor file. Isn't it confusing?

Reply:
It's in `e2e/` and it's for the End-to-end tests (tests that simulate a browser).

Unit tests will later go in `frontend/test`.

Reply:
Could it be called `e2e_tests` or something alike, then?